manifest.json

```
@@ -1,7 +1,7 @@
 {
   "name": "MyレコーダーChromeアシスタント",
   "short_name": "Myレコーダーアシスト",
-  "version": "0.9.2",
+  "version": "0.9.2.5",
```

@cti1650 ここはSemantic versioningにしておきたいです。
https://semver.org/lang/ja/
```suggestion
  "version": "0.9.3",
```

---

@shoito 
バージョンを指定頂ければそちらに修正します🙏